File: src/utils/response.rs
use serde::{Serialize, Deserialize};
use serde_json;

#[derive (Serialize, Deserialize)]
pub enum Response{
	Sucess(String), Error(String)
}

impl Response {
    pub fn from_string(data: String) -> Self {
        serde_json::from_str(data.as_str()).unwrap()
    }
    pub fn to_string(&self) -> String {
        serde_json::to_string(&self).unwrap()
    }
}

File: src/others/response.py
import json
from enum import IntEnum

class ResponseType(IntEnum):
    Sucess = 1
    Error = 2

class Response():
	def __init__(self, status: ResponseType, data):
		self.status = status.value
		self.data = data
        
	def to_string(self):
		return json.dumps(self.__dict__)
	
	staticmethod
	def from_string(data):
		obj = json.loads(data)
		status = ResponseType(int(obj['status']))
		return Response(status, obj['data'])  